src/messaging/ExchangeContext.h

```diff
@@ -214,6 +218,22 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext,
      */
     bool IsSendExpected() const { return mFlags.Has(Flags::kFlagWillSendMessage); }
 
+    /**
+     * Functions checks with information available in the ExchangeContext if the peer is active or not
```

Comment:
These API comments don't match the assumptions the implementation makes... though they do duplicate the implementation.  I would prefer the API comments explained how to use the method and when it's ok to call it instead of this.

Reply:
done